apps/demo/src/index.scss

```diff
@@ -64,7 +65,7 @@ html {
 .container {
   margin: auto;
   position: relative;
-  height: 100%;
```

Comment:
This currently breaks the layout on large screen. Indeed, the footer should be always sticked to the viewport bottom. Now, this is no more the case, the footer is displayed just under the data grid.